### DISCUSSION TITLE
I can't run it due to the lack of this library libcrypto.so.1.1

### DISCUSSION THREAD

**Discussion by lucasantoniooficial** [typegoose/mongodb-memory-server#812] [General]
How to solve the problem with this library? I can't find anything online about her.

**Comment by hasezoey** [comment]
i dont know what system or MMS version or Mongodb version you are running, but this question has been asked before:

- https://github.com/nodkz/mongodb-memory-server/issues/797
- https://github.com/nodkz/mongodb-memory-server/issues/732
- https://github.com/nodkz/mongodb-memory-server/issues/772

**Reply by lucasantoniooficial** [reply]
Do I need to have mongoDB installed on my machine? and that ?

Could I do this with a docker compose ?

**Reply by hasezoey** [reply]
i dont quite understand what you are asking now. MMS is a library that downloads mongodb binaries and manages starting / stopping a binary, it can also work with system binaries (binaries not downloaded by MMS).

please provide what system you are running (or if inside the container, the base image), which MMS version you are using and which Mongodb version is used(if not specified say default)
Note: "latest" is not a version

**Reply by lucasantoniooficial** [reply]
I haven't made a docker image yet, but my OS is a fedora budgie 38

**Reply by hasezoey** [reply]
there is no direct fedora build to my knowledge, the closed are rhel builds, could you try rhel82 and rhel90 builds? [archives can be found here](https://www.mongodb.com/download-center/community/releases/archive), this can be set via [DOWNLOAD_URL](https://nodkz.github.io/mongodb-memory-server/docs/api/config-options#download_url)(will likely also needs adjustment with [VERSION](https://nodkz.github.io/mongodb-memory-server/docs/api/config-options#version)) or manually download and extract the binaries and use [SYSTEM BINARY](https://nodkz.github.io/mongodb-memory-server/docs/api/config-options#system_binary)

currently MMS only supports up to rhel 8.2 and fedora 36

**Reply by lucasantoniooficial** [reply]
I would like Docker to be better for me, what would MMS be? A mongodb image?

**Reply by hasezoey** [reply]
> I would like Docker to be better for me, what would MMS be? A mongodb image?

i dont quite know what you are asking for, be we do have a [Integration with Docker](https://nodkz.github.io/mongodb-memory-server/docs/guides/integration-examples/docker) guide
